rpc/server.go

```diff
@@ -3,28 +3,31 @@ package rpc
 import (
 	"context"
 	"io"
-	"log"
 	"net"
 	"sync"
+
+	"github.com/pegasus-kv/meta-proxy/collector"
+	"github.com/sirupsen/logrus"
 )
 
 // Serve blocks until the connection shutdown.
 func Serve() error {
-	addr, err := net.ResolveTCPAddr("tcp", "0.0.0.0:34601")
+	addr, err := net.ResolveTCPAddr("tcp", "0.0.0.0:8888")
```

Comment:
The server port is important. It's where we should export to the users. Make it configurable, or don't change it.

Reply:
Oh, yes, it't changed since it is same with `pegasus_server onebox` port when I test,  it should be reverted,